package.json

```diff
@@ -34,7 +34,8 @@
     ],
     "moduleNameMapper": {
       "\\.(css|scss)$": "identity-obj-proxy"
-    }
+    },
+    "testURL": "http://localhost"
```

Comment:
This is needed due to a JSDom update that jest pulls in.
More info here: https://github.com/jsdom/jsdom/issues/2304